glyph/src/ttfp.rs

```diff
@@ -16,6 +16,44 @@ impl From<GlyphId> for owned_ttf_parser::GlyphId {
     }
 }
 
+/// A pre-rendered image of a glyph, usually used for emojis or other glyphs
+/// that can't be represented only using an outline.
+#[derive(Debug, Clone)]
+pub struct GlyphImage<'a> {
+    /// offset of the image from the normal origin (top at the baseline plus
+    /// ascent), measured in pixels at the image's current scale
+    pub origin: Point,
+    /// current scale of the image in pixels per em
+    pub scale: f32,
+    /// raw image data (not a bitmap)
+    pub data: &'a [u8],
+    /// format of the raw data
```

Comment:
Minor docs style stuff
```suggestion
    /// Offset of the image from the normal origin (top at the baseline plus
    /// ascent), measured in pixels at the image's current scale.
    pub origin: Point,
    /// Current scale of the image in pixels per em.
    pub scale: f32,
    /// Raw image data (not a bitmap).
    pub data: &'a [u8],
    /// Format of the raw data.
```